source/stylesheets/refills/_tables.scss

```diff
@@ -1,38 +1,21 @@
 .tables {
-  ///////////////////////////////////////////////////////////////////////////////////
-  $base-border-color: gainsboro !default;
-  $base-border-radius: 3px !default;
-  $base-line-height: 1.5em !default;
-  $base-spacing: 1.5em !default;
-  $base-accent-color: #477DCA !default;
-  $base-link-color: $base-accent-color !default;
-  $dark-gray: #333 !default;
-  $light-gray: #DDD !default;
-  $medium-screen: em(640) !default;
-  $large-screen: em(860) !default;
-  $base-font-color: $dark-gray !default;
-  
-  width: 100%;
-
-  th {
-    text-align: left;
-  }
-  //////////////////////////////////////////////////////////////////////////////////
-
+  $table-color: $base-font-color-on-box;
   $table-border-color: $base-border-color;
-  $table-border: 1px solid $table-border-color;
-  $table-background: lighten($table-border-color, 12%);
+  $table-border: $base-border;
+  $table-background: $base-box-background;
   $table-header-background: lighten($table-background, 10%);
   $table-hover-background: darken($table-background, 5%);
-  $table-stripe-background: darken($table-background, 2%);
+  $table-stripe-background: lighten($table-background, 2%);
   $table-stripe-background-hover: darken($table-stripe-background, 5%);
-  $table-padding: 0.75em 1em;
+  $table-padding: 0.75em $small-spacing;
 
+  color: $table-color;
```

Comment:
Properties should be ordered border, border-collapse, border-left, border-radius, border-spacing, color, width